cmd/validator/struct_processor.go

```diff
@@ -0,0 +1,466 @@
+package main
```

Comment:
This is the main comparison function. It serialized the responses into a struct and compares them field by field. The report on a difference is pretty nice and will tell you exactly which fields are different.